src/overlays/actors/ovl_Bg_Crace_Movebg/z_bg_crace_movebg.c

```diff
@@ -15,22 +16,28 @@ void BgCraceMovebg_Destroy(Actor* thisx, PlayState* play);
 void BgCraceMovebg_Update(Actor* thisx, PlayState* play);
 void BgCraceMovebg_Draw(Actor* thisx, PlayState* play);
 
-s32 func_80A7090C(PlayState* play);
-void func_80A70970(BgCraceMovebg* this, PlayState* play);
-void func_80A7099C(BgCraceMovebg* this, PlayState* play);
-void func_80A709E4(BgCraceMovebg* this, PlayState* play);
-void func_80A70A08(BgCraceMovebg* this, PlayState* play);
-void func_80A70A84(BgCraceMovebg* this, PlayState* play);
-void func_80A70A9C(BgCraceMovebg* this, PlayState* play);
-void func_80A70C04(BgCraceMovebg* this, PlayState* play);
-void func_80A70D74(BgCraceMovebg* this, PlayState* play);
-void func_80A70DA8(BgCraceMovebg* this, PlayState* play);
-void func_80A70E2C(BgCraceMovebg* this, PlayState* play);
-void func_80A70E70(BgCraceMovebg* this, PlayState* play);
-void func_80A70F14(BgCraceMovebg* this, PlayState* play);
-void func_80A70F2C(BgCraceMovebg* this, PlayState* play);
-void func_80A70FF4(BgCraceMovebg* this, PlayState* play);
-void func_80A71040(BgCraceMovebg* this, PlayState* play);
+s32 BgCraceMovebg_GetRaceStatus(PlayState* play);
+void BgCraceMovebg_OpeningDoor_SetupIdle(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_OpeningDoor_Idle(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_OpeningDoor_SetupOpen(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_OpeningDoor_Open(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_OpeningDoor_SetupDoNothing(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_OpeningDoor_DoNothing(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_CheckIfPlayerIsBeyondDoor(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_SetupIdle(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_Idle(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_SetupWaitToClose(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_WaitToClose(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_SetupClose(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_Close(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_SetupDoNothing(BgCraceMovebg* this, PlayState* play);
+void BgCraceMovebg_ClosingDoor_DoNothing(BgCraceMovebg* this, PlayState* play);
+
+typedef enum {
+    /* 0 */ BG_CRACE_MOVEBG_RACE_STATUS_BUTLER_NOT_PRESENT,
+    /* 1 */ BG_CRACE_MOVEBG_RACE_STATUS_RACE_NEVER_STARTED,
+    /* 2 */ BG_CRACE_MOVEBG_RACE_STATUS_RACE_STARTED_ONCE_BEFORE,
```

Comment:
```suggestion
    /* 2 */ BG_CRACE_MOVEBG_RACE_STATUS_RACE_STARTED_ONCE_BEFORE
```
We have `-woff 624`, but better to just remove them in the first place.